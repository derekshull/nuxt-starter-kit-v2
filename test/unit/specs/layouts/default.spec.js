import { shallowMount } from "@vue/test-utils";
import Layout from "~/layouts/default.vue";

describe("Component Button", () => {
  it("renders to a snapshot", () => {
    const wrapper = shallowMount(Layout, {
      stubs: ["nuxt"]
    });
    expect(wrapper.element).toMatchSnapshot();
  });
});
