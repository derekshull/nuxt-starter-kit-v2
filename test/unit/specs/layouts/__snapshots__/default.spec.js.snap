// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Button renders to a snapshot 1`] = `
<div>
  <nuxt-stub />
   
  <myfooter-stub />
</div>
`;
